README.md

```diff
@@ -42,6 +42,61 @@ Data import is still a fairly manual process leveraging the DB and a few SQL scr
 
 *Remember to run `VACUUM ANALYSE` or REINDEX tables after significant changes to ensure stats are up to date to use indices properly.*
 
+# Standard Operating Procedure
+
+This covers how to keep the data up to date. Each quarter, as new data is released, the following needs to be done to update the data served by the API and the Citizen Scorecard. It's best to do this on a test database first and verify the results before updating the production database.
+
+## Extract CSV datasets from Excel Spreadsheets
```

Comment:
These scripts are necessary to produce the CSVs for the non-financials. They should be updated using similar scripts but I didn't write them now because I don't have new data for them.